### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#381

### CODE REVIEW

**Review by mingazhev — COMMENTED** [review]
Need some more research on samplers

File: HappyTravel.Telemetry/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+namespace HappyTravel.Telemetry.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddTracing(this IServiceCollection services, Action<TelemetryOptions> action)
+        {
+            var options = new TelemetryOptions();
+            action.Invoke(options);
+
+            if (!options.IsEnabled)
```
Comment (kirillta) [inline HappyTravel.Telemetry/Extensions/ServiceCollectionExtensions.cs:17]: Why do we have this one? One always may simply comment the method

File: nuget.config [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="github" value="https://nuget.pkg.github.com/happy-travel/index.json" />
```
Comment (mingazhev) [inline nuget.config:4]: Don't we need nuget there?

**Review by mingazhev — APPROVED** [review]
With a little comment

File: nuget.config (hunk 2 above)
Comment (DFSko) [reply nuget.config:4]: nuget feed enabled by default

File: HappyTravel.Telemetry/Extensions/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.Telemetry/Extensions/ServiceCollectionExtensions.cs]: Removed this condition

**Pull request merged**